go-packages/meep-redis/db.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/go-redis/redis"
 )
 
+const dbMaxRetryCount = 2
```

Comment:
suggest adding a resting time between each retry

Reply:
The error occurs on a connection timeout, therefore we've already waited for some period of time before retrying. Worst case scenario the process fails 3 times in a row and exits, at which point the pod restart & backoff timers kick in. We should be okay without adding an additional timer here for now, unless an obvious issue appears where we would benefit from it.